### ISSUE TITLE
Capture certificate expiration and issuer in diagnostics

### ISSUE THREAD

**Issue by mcpate** [elastic/support-diagnostics#464]
We recently had an SDH that was a bit complicated to troubleshoot. Eventually, we realized that the internal certificate expired which was causing the Proxy to reject requests from our internal services. It would be really nice if the diagnostics tool captured the expiration of the internal certs and the issuer so that we could check those things as top-level items when troubleshooting ECE issues. Having the certificate expiration in the ECE diagnostics might not always catch the problem (there are certain scenarios where _some_ services might be using an expired certificate and _some_ might not be), but in the cases where it does catch it we could save a lot of time troubleshooting.

The following is an example of how we ran `openssl` on the underlying host to verify the certificate expiration:
```
openssl x509 -in /mnt/data/elastic/ece-0-runner/services/route-server/managed/cert.pem -inform pem -noout -enddate cert.pem ca-file.pem
```

The following is an example of how we ran `openssl` on the underlying host to verify the issuer (which highlights whether the cert is our internal cert or one that the customer uploaded):
```
openssl x509 -in /mnt/data/elastic/ece-0-runner/services/route-server/managed/cert.pem -inform pem -noout -issuer
```

**Comment by mcpate** [comment]
Wrong repo.
